site/content/faq/why-does-my-antivirus-tool-flag-zap.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Why does my Antivirus Tool Flag ZAP?"
+type: faq
+category: General Questions
+weight: 2
+---
+
+We know that many Antivirus (AV) tools flag ZAP and some of the ZAP add-ons.
+
+For example the ZAP 2.15 Windows installer was 
+[flagged by 3 / 63](https://www.virustotal.com/gui/file/28b348dd65116ddabbbbd98b7f84864a0bb0f98d656266f2f08bfd010ae51c57)
+security vendors.
+
+In particular the [Active Scan Rule add-on](/docs/desktop/addons/active-scan-rules/) is often flagged: v65 was 
+[flagged by 10 / 63](https://www.virustotal.com/gui/file/84b91ad0dc613838ab57b5f78fb091d63026cef77c2bf223ea68e97fa05b4181)
+security vendors.
+
+Detecting viruses is hard, especially as viruses try to disguise themselves.
+This means that AV tools try to detect potentially malicious activity or code.
+
+ZAP is a security tool which "does bad things".
+
+The [Active Scan Rule add-on](/docs/desktop/addons/active-scan-rules/) contains the rules which attack websites, 
+so it is not surprising that make AV tools flag it.
+
+Any issues raised about ZAP or its add-ons being flagged by AV tools will be closed with a link to this FAQ.
+
+If you have hard evidence that there is an actual virus in ZAP or its add-ons then please report that with the evidence to 
+the ZAP [Bug Bounty Program](https://bugcrowd.com/owaspzap).
+
+If you work for an AV tool vendor and would like to discuss how you can make sure your tool does not incorrectly flag ZAP then please
+[get in touch](mailto:support@zaproxy.com).
```

Comment:
I'd say move this up to line 10 😀

Reply:
I know what you mean, but the "For example.." paragraph doesnt make as much sense then?

Reply:
Move it to the very top and put a heading in before "we know", or even turn "we know" into like a 2, 3, 4 level heading?

Reply:
The main target audience for this FAQ will be ZAP users. How about I leave this line where it is but make it bold, or a heading?

Reply:
Sure!

Reply:
Made it **bold** :)